Assets/Scripts/Actors/ActorPhysics.cs

```diff
@@ -343,7 +343,13 @@ public void AirMovement( Vector3 inputVec, bool forceUp = false )
 		}
 		else
 		{
-			_moveVec = inputVec * _jumpMoveSpeed;
+			_moveVec += inputVec * _jumpMoveSpeedChangeRate;
+			_moveVec.y = 0.0f;
+
+			if ( _moveVec.sqrMagnitude > _sprintMoveSpeed * _sprintMoveSpeed )
+			{
+				_moveVec = _moveVec.normalized * _sprintMoveSpeed;
+			}
```

Comment:
This might be a game feel thing but why isn't this being normalized to the sprint speed or grounded speed based on button input?